RecoLocalTracker/SiPixelRecHits/interface/SiPixelGenError.h

```diff
@@ -104,9 +119,10 @@ class SiPixelGenError {
 public:
    SiPixelGenError(const std::vector< SiPixelGenErrorStore > & thePixelTemp) : thePixelTemp_(thePixelTemp) { id_current_ = -1; index_id_ = -1;} //!< Constructor for cases in which template store already exists
    
-   static bool pushfile(int filenum, std::vector< SiPixelGenErrorStore > & thePixelTemp_);     // load the private store with info from the
-   // file with the index (int) filenum
-   
+// Load the private store with info from the file with the index (int) filenum from directory dir:
+//   ${dir}generror_summary_zp${filenum}.out
+   static bool pushfile(int filenum, std::vector< SiPixelGenErrorStore > & thePixelTemp_ , std::string dir = "");
```

Comment:
I notice that this argument ```thePixelTemp_ ``` is exactly the same spelling as the data member of the class .
This is at least confusing.
Please drop the underscore in the function argument or change the data member.